src/linear_algebra/basic.lean

```diff
@@ -56,303 +56,305 @@ open function lattice
 reserve infix ` ≃ₗ `:25
 
 universes u v w x y z
-variables {α : Type u} {β : Type v} {γ : Type w} {δ : Type y} {ε : Type z} {ι : Type x}
+variables {R κ : Type u} {M V : Type v} {M₂ V₂ : Type w} {M₃ V₃ : Type y} {M₄ : Type z} {ι : Type x}
```

Comment:
Why are there variables in the same universe, instead of one universe for each variable? This seems to be calling for trouble...

Reply:
They should never mix. `R` replaced `α` in module defs/theorems, `κ` for vector spaces.

Reply:
Until someone writes in this file a statement with `R` and `κ` and doesn't pay attention to the fact that they belong to the same universe...

Reply:
Oh, fair enough. I was thinking in terms of faithfully translating the old file. Updated -- this should be safe in both respects.